framework/resources/Functional/decimal_parquet/InternalCastLiteralAsDecimalWhenDecimalsOn.sql

```diff
@@ -0,0 +1,6 @@
+SET `planner.enable_decimal_data_type` = true;
```

Comment:
What is the system default for this setting? What is it set in our before_run.sql? What a RESET does? Do you see potential conflict? Thanks.